pkg/cmd/provisioning/nutanix/create_shared_secrets_test.go

```diff
@@ -0,0 +1,199 @@
+package nutanix
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testCredReqDirPrefix     = "nutanix_test_dir_credreq"
+	testTargetDirPrefix      = "nutanix_test_dir_target"
+	testCredentialsDirPrefix = "nutanix_test_dir_credentials"
+
+	credReqTemplate = `---
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: %s
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: %s
+  secretRef:
+    name: %s
+    namespace: %s`
+
+	basicAuthCredentialsTemplate = `
+credentials:
+- type: basic_auth
+  data:
+    prismCentral:
+      username: %s
+      password: %s`
```

Comment:
Should it also include auth for prism elements as defined [here](https://github.com/openshift/cloud-credential-operator/blob/master/pkg/cmd/provisioning/nutanix/create_shared_secrets.go#L48-L56)? I understand it won't affect the tests here but just for consistency and correctness.

Reply:
Those are optional and currently, not in use. Let me see, I can add a test that tests the contents of the output too. Then we can have one with and one without.